.github/workflows/build_release.yml

```diff
@@ -0,0 +1,87 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Publish release image to GitHub Container Registry
+
+on:
+  push:
+    tags: [ "v[0-9].[0-9].[0-9]" ]
+  workflow_dispatch:
```

Comment:
looks like we don't need this. remove?

Reply:
`workflow_dispatch`? It's only to be able to launch the job manually, should I remove?

Reply:
Removed for release